src/components/layouts/app-layout.tsx

```diff
@@ -60,6 +70,20 @@ const AppLayout: React.VoidFunctionComponent<Props> = ({ drawerContent, children
     setSyncingData(false);
   };
 
+  /**
+   * Handler for sync-date selection dialog
+   */
+  const handleClickOpen = () => {
+    setSyncSelection(true);
+  };
+
+  /**
+  * Handler for sync start date
+  */
+  const resetSyncDate = () => {
+    setSelectedStartDate(null);
+  };
```

Comment:
Remove these and update the state inline where these function are called.

Reply:
Done.